samples/electron-app/src/main/main.ts

```diff
@@ -0,0 +1,55 @@
+import { app, BrowserWindow } from 'electron';
+import * as path from 'path';
+import { ElectronAuthenticator } from '@microsoft/mgt-electron-provider/dist/Authenticator';
+export default class Main {
+  static application: Electron.App;
+  public static mainWindow: Electron.BrowserWindow;
+  static authProvider: ElectronAuthenticator;
+
+  static main(): void {
+    Main.application = app;
+    Main.application.on('window-all-closed', Main.onWindowAllClosed);
+    Main.application.on('ready', Main.onReady);
+  }
+  static onWindowAllClosed(arg0: string, onWindowAllClosed: any) {
+    Main.application.quit();
+  }
+
+  static onReady() {
+    Main.createMainWindow();
+    Main.authProvider = new ElectronAuthenticator({
```

Comment:
Looks like `Main.authProvider` is not used anywhere. I proposed we change the usage of ElectronAuthenticator to act more like a service instead of an instance. What do you think about changing the usage to:

```
ElectronAuthenticator.initialize({...});
```

Calling initialize would initialize the service for the entire app globally, and the developer won't have to keep track of the instance. Calling the `initialize` static method a second time will not have an effect. If the developer wants the instance of the authenticator for whatever reason, we could add an `instance` static field:

```
let authenticator = ElectronAuthenticator.instance
```

Thoughts?